spec/presenters/presenter_spec.rb

```diff
@@ -20,4 +20,19 @@
       expect(subject.salary_frequency_options).to eq(expected_result)
     end
   end
+
+  describe '#formatted_currency' do
+    let(:number) { 5_876 }
+    context 'when a precision is provided' do
+      it 'formats a number with a "£" the given number of decimal places' do
+        expect(subject.formatted_currency(5_876, 0)).to eq('£5,876')
+      end
+    end
+
+    context 'when NO precision is provided' do
```

Comment:
Does this, in essence, mean default precision? If so should we word the spec to reflect this?